app/javascript/components/EditForm.js

```diff
@@ -76,33 +94,43 @@ class EditRebateForm extends React.Component {
             <div className="flex-row">
               {map(customerDetailFields, (field) => {
                 return field.type == 'radio'
-                ? RadioInput(field)
-                : SingleInput(field)
+                ? RadioInput({...field, isEditable})
+                : SingleInput({...field, isEditable})
               })}
             </div>
             { values.fields.moved_within_rating_year == 'yes'
```

Comment:
For conditional rendering of components, I prefer to use `{ <condition> && <component /> }` over the ternary operator